lib/error/errors.go

```diff
@@ -50,4 +50,7 @@ var (
 	ErrorBallotAlreadyVoted                   = NewError(143, "ballot already voted")
 	ErrorBallotHasOverMaxTransactionsInBallot = NewError(144, "too many transactions in ballot")
 	ErrorMessageHasIncorrectTime              = NewError(145, "time in message is not correct")
+	ErrorFrozenAccountNoDeposit               = NewError(146, "frozen account can not receive payment")
```

Comment:
please add items for map even if all of the values are 400
https://github.com/bosnet/sebak/blob/76c50e8ca9c3151fd9112334bf38a3879bd4f056/lib/network/httputils/httputils.go#L18

The status code will be changed later.